host_source.go

```diff
@@ -157,6 +157,7 @@ type HostInfo struct {
 	state            nodeState
 	schemaVersion    string
 	tokens           []string
+	isRackNil        bool
```

Comment:
We can change `rack` to be `*string`, it's a private field. A conversion has to happen in the `Rack()` function though.

Reply:
Sounds good to me!
Let's wait for the feedback that you have asked for.